src/celeritas/random/distribution/RejectionSampler.hh

```diff
@@ -62,7 +62,9 @@ class RejectionSampler
 
     //! Construct when the distribution's maximum is normalized
     explicit CELER_FUNCTION RejectionSampler(real_type f)
-        : RejectionSampler{f, 1} {}
+        : RejectionSampler{f, 1}
+    {
+    }
```

Comment:
Looks like someone *ahem* 😳 forgot to clang-format their PR and ended up making everyone else's PRs harder. Perhaps I should run a big clang-format now?